Modules/include/ConfigReader.h

```diff
@@ -232,40 +235,57 @@ namespace ChimeraTK {
   /********************************************************************************************************************/
 
   template<typename T>
-  const T& ConfigReader::get(const std::string& variableName, const T& defaultValue) const {
-    /// FIXME: Do not implement with try-catch.
-    try {
-      return getImpl(variableName, static_cast<T*>(nullptr));
+  const T& ConfigReader::get(std::string variableName, const T& defaultValue) const {
+    if(variableName.starts_with("/")) {
+      variableName = variableName.substr(1);
     }
-    catch(ChimeraTK::logic_error&) {
+    auto result = getImpl(variableName, static_cast<T*>(nullptr));
+    if(!result.has_value()) {
```

Comment:
I think you can just `return *result.or(&defaultValue)` or something in that direction